tools/tpm2_import.c

```diff
@@ -870,12 +870,12 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
       { "parent-key",         required_argument, NULL, 'C'},
       { "parent-key-public",  required_argument, NULL, 'K'},
       { "import-key-private", required_argument, NULL, 'r'},
-      { "import-key-public",  required_argument, NULL, 'q'},
+      { "import-key-public",  required_argument, NULL, 'u'},
       { "object-attributes",  required_argument, NULL, 'A'},
       { "halg",               required_argument, NULL, 'g'},
     };
 
-    *opts = tpm2_options_new("G:k:C:K:q:r:A:g:", ARRAY_LEN(topts), topts, on_option,
+    *opts = tpm2_options_new("G:k:C:K:u:r:A:g:", ARRAY_LEN(topts), topts, on_option,
                              NULL, 0);
```

Comment:
Need to update the tpm2_import manual page for this change

Reply:
Good catch, I saw your comment and I was like no i did that...nope I updated the changelog...